custom_components/zwave_mqtt/entity.py

```diff
@@ -157,7 +158,11 @@ def device_info(self):
     @property
     def device_state_attributes(self):
         """Return the device specific state attributes."""
-        attrs = {const.ATTR_NODE_ID: self.values.primary.node.node_id}
+        attrs = {
+            const.ATTR_NODE_ID: self.values.primary.node.node_id,
+        }
+        if self.power_consumption is not None:
```

Comment:
We are calculating `power_consumption` twice, since it's a calculated property. 

Also, you're adding this to the generic Z-Wave entity. Is this on purpose ?

Reply:
In what other place is the power_consumption calculated ? 
And yes, Added it on purpose to the generic entity, so it will be picked up by every (future) device implementation. Currently supported by switches and lights as far as I can see.

Looks like the previous/current z-wave implementation uses the same approach.

Reply:
@balloob any word on this ?